app/Controller/SongsController.php

```diff
@@ -194,8 +194,8 @@ public function albums() {
                 'Song' => array(
                     'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
                     'conditions' => $subQuery,
-                    'order' => 'Song.album',
-                    'limit' => 36
+                    'order' => ['Song.band', 'Song.album'],
+                    'limit' => 60
```

Comment:
I think the proper fix is this one: https://github.com/cakephp/cakephp/issues/7324#issuecomment-201211755

There is a bug when CakePHP is rebuilding the URL for the next page of the paginator. Apparently the key line in this comment is this:
> The function key() and current() are used to set the sorting, completely ignoring the possibility of multiple sort fields.

Then he apparently gives the solution to fix it.

I'm at work right now so I can't test it, but I will as soon as I'm home.